app/assets/images/decidim-logo.svg

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The logo and the badge are already in decidim, I guess we don't need to add them here again. Can you confirm @isaacmg410 ?

Reply:
Done https://github.com/gencat/participa/pull/108/commits/f8c269d54540d5f4ae76612d457834d129dee0db

Reply:
Done https://github.com/gencat/participa/pull/108/commits/f8c269d54540d5f4ae76612d457834d129dee0db